modules/ROOT/pages/mule-application-about.adoc

```diff
@@ -33,6 +35,8 @@ For example, the Salesforce Connector provides components that let you use the S
 .The Salesforce connector and its components listed in Anypoint Studio
 image::about-mule-apps-salesforce-connector.png[The Salesforce connector and its components listed in Anypoint Studio]
 
+For connector documentation, see xref:connectors::index.adoc[Connectors and Modules (for Mule 4)].
+
 == Modules
 
 Modules group together components that were created to add flexibility to your apps by allowing you to aggregate values, compress data, use Java features, use extra features for processing JSON, and more. The features that modules offer are more at the level of traditional application coding.
```

Comment:
I don't know what components means outside of Core Components.

Reply:
Each box or card in a flow can be generically called a [Mule Component](https://docs.mulesoft.com/mule-runtime/3.9/components). In Mule 4 Core components are the ones built-in in Mule.